src/content/content.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@nestjs/common'
+import { CreateContentDto } from './dto/create-content.dto'
+import { UpdateContentDto } from './dto/update-content.dto'
+
+@Injectable()
+export class ContentService {
+  create(createContentDto: CreateContentDto) {
+    console.log(`createContentDto =>`, createContentDto)
+    return 'This action adds a new content'
+  }
+
+  findAll() {
+    return `This action returns all content`
+  }
+
+  findOne(id: number) {
+    return `This action returns a #${id} content`
+  }
+
+  update(id: number, updateContentDto: UpdateContentDto) {
+    console.log(`updateContentDto =>`, updateContentDto)
```

Comment:
I added these console logs just to appease eslint for now from complaining about the arguments not being used.